Sources/Proxy/Core/View2DProxy/index.js

```diff
@@ -21,6 +23,38 @@ function formatAnnotationValue(value) {
   return value;
 }
 
+/**
+ * Returns an array of points in world coordinates creating a coarse hull
+ * around the prop given in argument
+ * The returned array is empty if the prop is not visible or doesn't use bounds
+ *
+ * How it works: if possible, combine the mapper bounds corners with the prop matrix
+ * otherwise, returns the prop bounds corners
+ */
+function getPropCoarseHull(prop) {
+  if (!prop.getVisibility() || !prop.getUseBounds()) {
+    return [];
+  }
+  let mapperBounds;
+  if (prop?.getMapper) {
+    const mapper = prop.getMapper();
+    if (mapper?.getBounds) {
+      mapperBounds = mapper.getBounds();
+    }
+  }
```

Comment:
`const mapperBounds = prop?.getMapper.?()?.getBounds.?()`

Reply:
![image](https://user-images.githubusercontent.com/90781029/215836629-f655887f-8299-4aa5-a075-c7aebfae884b.png)

It's working in the console but eslint doesn't want me to do it

Reply:
Should I leave it as is?

Reply:
Nevermind, it is `?.` and not `.?`